modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/partner-portal/fragments/group/clay-icon-link/fragment.json

```
@@ -0,0 +1,9 @@
+{
+	"configurationPath": "configuration.json",
+	"cssPath": "index.css",
+	"htmlPath": "index.html",
+	"icon": "code",
+	"jsPath": "index.js",
+	"name": "ClayIcon",
```

O nome do fragment que está na pasta é "clay-icon-link" e o name está "ClayIcon". É importante que essas informações batam, então seria interessante ou alterar o nome da pasta para clay-icon e adicionar esse espaço no name, ou seja, "Clay Icon". Ou alterar o name para ficar "Clay Icon Link".

```suggestion
	"name": "Clay Icon Link",
```